tests/lib/lodSelection.cpp

```diff
@@ -66,7 +66,7 @@ Identifiers getVisibles( const livre::DataSource& dataSource,
                                           screenSpaceError,
                                           minLOD,
                                           maxLOD,
-                                          { 0.0f, 1.0f } );
+                                          {{ 0.0f, 1.0f }});
```

Comment:
../tests/Lib/lodSelection.cpp:69:45: error: suggest braces around initialization of subobject [-Werror,-Wmissing-braces]